pyproject.toml

```diff
@@ -46,11 +46,12 @@ pyright = "^1.1.270"
 pytest-cov = "^4.0.0"
 starknet-devnet = "^0.4.4"
 isort = "^5.11.4"
+pytest-rerunfailures = "^11.1"
 
 [tool.poe.tasks]
-test.shell = "pytest -n auto -v --cov=starknet_py starknet_py"
-test_ci.shell = "coverage run -m pytest starknet_py --ignore=starknet_py/tests/e2e/docs"
-test_ci_docs.shell = "coverage run -m pytest starknet_py/tests/e2e/docs"
+test.shell = "pytest -n auto -v --reruns 10 --only-rerun aiohttp.client_exceptions.ClientConnectorError --cov=starknet_py starknet_py"
+test_ci.shell = "coverage run -m pytest -v --reruns 2 --only-rerun aiohttp.client_exceptions.ClientConnectorError starknet_py --ignore=starknet_py/tests/e2e/docs"
+test_ci_docs.shell = "coverage run -m pytest -v --reruns 2 --only-rerun aiohttp.client_exceptions.ClientConnectorError starknet_py/tests/e2e/docs"
```

Comment:
I think I'd avoid automatic reruns on CI. Failures due to aiohttp errors in non-parallel tests are not frequent.